fairseq-sl/fairseq/data/token_block_utils_fast.cpp

```diff
@@ -1,28 +1,25 @@
-/* Generated by Cython 3.0.6 */
+/* Generated by Cython 3.0.11 */
 
 /* BEGIN: Cython Metadata
 {
     "distutils": {
         "depends": [
-            "/tmp/pip-build-env-zoh18iqa/overlay/lib/python3.10/site-packages/numpy/core/include/numpy/arrayobject.h",
-            "/tmp/pip-build-env-zoh18iqa/overlay/lib/python3.10/site-packages/numpy/core/include/numpy/arrayscalars.h",
-            "/tmp/pip-build-env-zoh18iqa/overlay/lib/python3.10/site-packages/numpy/core/include/numpy/ndarrayobject.h",
-            "/tmp/pip-build-env-zoh18iqa/overlay/lib/python3.10/site-packages/numpy/core/include/numpy/ndarraytypes.h",
-            "/tmp/pip-build-env-zoh18iqa/overlay/lib/python3.10/site-packages/numpy/core/include/numpy/ufuncobject.h"
+            "C:\\Users\\myous\\AppData\\Local\\Temp\\pip-build-env-7b3b5wvo\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h",
+            "C:\\Users\\myous\\AppData\\Local\\Temp\\pip-build-env-7b3b5wvo\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h",
```

Comment:
I don't think changing a Unix-dependent (yet user-neutral) to a Windows-dependent (and user-specific) is a better refactoring choice

Reply:
retrieved them. Done